app/views/common.py

```
@@ -99,6 +101,25 @@ def travel_content_view(content: str):
+# MARK: notice
+def notice_view():
+    """
+    お知らせコンテンツを表示する。
+    """
+    notice, timestamp_str = spreadsheet_service.get_notice()
+    if notice == "" or timestamp_str == "":
+        return jsonify({"notice": "", "timestamp": ""})
+
+    timestamp_datetime = datetime.strptime(timestamp_str, "%m/%d/%Y %H:%M:%S")
+
+    # 日本時間（JST）として扱う
```

Ruff (0.14.3)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)